### ISSUE TITLE
Publish on F-Droid

### ISSUE THREAD

**Issue by jubalh** [chickendude/Glossika-Android#52]
Have you considered publishing this on F-Droid? Might make it easier for users to install/update the app.

**Comment by chickendude** [comment]
Yeah i originally had planned to publish to F-Droid and the Play Store when it was in a more stable state (and i don't have to worry so much about changing the database scheme), but it is pretty niche and i'm not sure how many people would be able to find use out of it on their own without the other scripts. F-Droid might be a better platform than the Play Store for now though. I'll consider putting up a release there.

I'd like to figure out a decent way to add your own sentences so you can more easily create your own courses, but i haven't come up with anything specific yet.

**Comment by jubalh** [comment]
An F-Droid release would it make it easier for me to update the app.
Currently I have to navigate with my browser to the repo url, go to releases, grab the APK, install it manually...
F-Droid would be much more comfortable.
